build/templates/End_To_End_Tests_On_Windows.yml

```diff
@@ -67,7 +67,7 @@ steps:
   inputs:
     scriptName: "$(System.DefaultWorkingDirectory)\\artifacts\\EndToEnd\\scripts\\RunFunctionalTests.ps1"
     arguments: "-PMCCommand $(EndToEndTestCommandToRunPart) -PMCLaunchWaitTimeInSecs 30 -EachTestTimoutInSecs 600 -NuGetDropPath $(System.DefaultWorkingDirectory)\\artifacts -FuncTestRoot $(System.DefaultWorkingDirectory)\\artifacts -RunCounter $(Build.BuildNumber) -VSVersion 16.0"
-  condition: "and(succeeded(), eq(variables['IsOfficialBuild'], 'true'))"
+  condition: "and(succeeded(), not(eq(variables['IsOfficialBuild'], 'true')))"
```

Comment:
the `continueOnError: "true"` on line 66 needs to be deleted.

Reply:
Thanks. Changed continueOnError to "false" in line 66.